pydantic_ai_slim/pydantic_ai/providers/google_gla.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations as _annotations
+
+import os
+
+import httpx
+
+from pydantic_ai.models import cached_async_http_client
+from pydantic_ai.providers import Provider
+
+
+class GoogleGLAProvider(Provider[httpx.AsyncClient]):
+    """Provider for Google Generative Language AI API."""
+
+    @property
+    def name(self):
+        return 'google-gla'
+
+    @property
+    def base_url(self) -> str:
+        return 'https://generativelanguage.googleapis.com/v1beta/models/'
+
+    @property
+    def client(self) -> httpx.AsyncClient:
+        return self._client
+
+    def __init__(self, api_key: str | None = None, http_client: httpx.AsyncClient | None = None) -> None:
+        api_key = api_key or os.environ.get('GEMINI_API_KEY') or os.environ.get('GOOGLE_GLA_API_KEY')
```

Comment:
I've added the new `GOOGLE_GLA_API_KEY`, should I remove it?

Reply:
Everywhere else we just use environment variable names we didn't come up with; I would suggest we do that here (i.e., whatever the google SDKs use, we should use. I recognize we don't have to do that, but 🤷‍♂️ )